docs/content/docs/troubleshooting/upgrade.md

```diff
@@ -1,3 +1,9 @@
+---
+title: "Upgrade Troubleshooting"
+description: "OSM Upgrade Troubleshooting Guide"
+type: docs
```

Comment:
Do you need `aliases: ["troubleshooting"]` in this section?

Reply:
👍 

Good idea to make the previous url point to the new one.

Reply:
@flynnduism - could you break down what aliases are for us? Are these the tags that folks can use for search purposes in the UI?

Reply:
Sure thing! [Aliases](https://gohugo.io/content-management/urls/#aliases) are Hugo's way to handle content redirects.

So here, the upgrade page exists at `/troubleshooting/upgrade`. If the page had previously had a diffferent url structure (like `troubleshooting-upgrades` or something) the addition of an alias would create additional pages in thge Hugo site that redirect to this page.

```
alias: ["troubleshooting-upgrades"]
```

Taking another look at this PR I see the page is being moved into the docs structure, but there isn't really a path change so I don't think an alias is needed in this instance